docs/sphinx/examples/cpp/basics/noise_phase_flip.cpp

```diff
@@ -20,11 +20,11 @@ int main() {
   // our phase flip channel to.
   cudaq::noise_model noise;
 
-  // Phase flip channel with `1.0` probability of the qubit
-  // undergoing a phase rotation of 180 degrees (π).
+  // We define a phase-flip channel setting to `1.0` the probability of the
```

Comment:
nit: Like Eric, I personally find the "... setting the probability of ... to 1.0" much more natural to read.

Reply:
the last commit fixes this.